hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,3 @@
+RELEASE_TYPE: minor
+
+This release has deprecated the generation of integers, floats and fractions when the conversion of the upper and/ or lower bound is not 100% exact, e.g. when an integer gets passed a bound that is not a whole number. (:issue:`1625`)
```

Comment:
Break lines at 80 characters.

We also like to have a thankyou note in the changelog, so a final line "Thanks to Felix Grünewald for this patch." would be great too (but optional).